### ISSUE TITLE
xlwings addin not working properly - modules not found

### ISSUE THREAD

**Issue by joancj** [xlwings/xlwings#876]
I've posted this issue in stackoverflow, let's see if I am more lucky here:

I've been trying to install and use the xlwings addin for several days now but I am not able to get it running... I really would like to use xlwings, but it is being painfully difficult.
I am using Anaconda.

xwlings is installed and works perfectly (I can read/write from/to an .xlsx file) but what I want to really do is to call python functions from excel and for that I need the addin.
The docs are not super useful here and I've tried to follow this guide (from https://newtonexcelbach.com/2017/07/30/installing-xlwings-0-11-4/):

" Run xlwings addin install from a command prompt. Reopen Excel and check if the xlwings Ribbon appears. If not, copy xlwings.xlam (from your xlwings installation folder under addin\xlwings.xlam manually into the XLSTART folder. You can find the location of this folder under Options > Trust Center > Trust Center Settings… > Trusted Locations, under the description Excel default location: User StartUp. Restart Excel and you should see the add-in"

When I opened excel I did not see the addin. I put the xlwings .xlam file into the XLSTAR folder... reopened excel and the addin was still not there. Then I put the .xlam file also in this folder : "C:\Users\UserName\AppData\Roaming\Microsoft\AddIns". The .xlam file is in both folders, and now I can see the xlwings addin (later I've removed the .*xlam file from the \Microsoft\AddIns so now it is only installed in the XLSTAR folder)

then I create a folder with myproject.xlsm and myproject.py files by running "xlwings quickstart myproject" from the anaconda promt. The files are there but then when I try to call any python function/Script from the vba modules I always get this error:

"Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "c:\users\joanc\myproject\myproject.py", line 1, in <module>
    import xlwings as xw
ModuleNotFoundError: No module named 'xlwings'"

for example, I get this error when I try to run the code that is already available when myproject.xlsm is created. The code looks like this:
sub SampleCall()
    mymodule=Left(ThisWorkbook.Name, (InStrRev(ThisWorkbook.Name, ".",-1, vbTextCompare)
    RunPython("import " & mymodule & ";" & mymodule ".hello_xlwings()")
end sub

I tried to run this code directly from the developer tab and also from a button created in sheet1 and assigning its corresponding macro...

As a side note if I go to the developer tab>tools>references i can see the xlwings checkbox active... so this should not be a problem.

I've downloaded the Montecarlo example and when I run it i get the following error:
Traceback (most recent call last): File "<string>", line 1, in <module> File "c:\users\joanc\desktop\montecarlo\simulation.py", line 9, in <module> import numpy as np ModuleNotFoundError: No module named 'numpy'". 

So it looks like excel is connecting to the python script but does not found the modules... maybe this helps to solve the problem 

Well, this is all I've done so far... no idea what I am doing wrong and what to do next. I would really like to use this package but if anyone knows a good alternative I am totally open (not only to read/write from/to excel but also to call python scripts/functions directly from excel)
Thank you very much for your help

**Comment by DougAJ4** [comment]
Which version of excel are you using?

If 64 bit, this may be relevant:
https://github.com/ZoomerAnalytics/xlwings/issues/820

**Comment by joancj** [comment]
Hi, excel is 32 bits

the exact version is:
Microsoft Excel 2016 MSO (16.0.9226.2156) 32 bits

hope it helps,

thank you very much

**Comment by DougAJ4** [comment]
I can't be of much help I'm afraid.  I can only suggest:

1. Use the on-line documentation, rather than the download pdf.  (I wrote the blog article you linked in your first post, but that was basically copy and pasted from the on-line docs).
2. Check your path 
3. Search both open and closed issues here for similar problems.

**Comment by DougAJ4** [comment]
Coincidentally I have had to update my xlwings installation on two machines over the past few days.  One updated with no problems, but the other gave problems similar to yours.

The problem was I had two users defined, and I installed Anaconda, including xlwings, under one user's path, and my Excel files under the other user's path.  Reinstalling Anaconda under the same user as the Excel files solved the problem.

Could that be your problem?
